tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss

```diff
@@ -34,259 +28,3 @@ $uxpl-light-blue-base: rgb(242, 248, 251) !default;
 $uxpl-green-base: rgb(21, 55, 109) !default; // wcag2a compliant
 $uxpl-green-hover-active: lighten($uxpl-green-base, 8%) !default; // wcag2a compliant
 $uxpl-green-dark-hover-active:  lighten($uxpl-green-base, 8%) !default;
-
-$uxpl-pink-base: rgb(194, 56, 125) !default; // wcag2a compliant
-$uxpl-pink-hover-active: lighten($uxpl-pink-base, 7%) !default; // wcag2a compliant
-
-$uxpl-grayscale-x-back: rgb(245, 245, 245) !default; // UXPL grayscale, x-back
-
```

Comment:
I assume all of these variables are now coming from cms/static/sass/partials/cms/theme/variables-v1?

Reply:
nvm, got my answer from https://github.com/overhangio/tutor-indigo/blob/nightly/tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss

Reply:
Yes,  add import `cms/static/sass/partials/cms/theme/variables-v1` at the top. Only those variables exists that we've overrided.